sm/mprv.h

```diff
@@ -2,37 +2,52 @@
 #include <stdint.h>
 #include <stdbool.h>
 
+typedef struct { size_t words[8]; } mprv_block_t;
```

Comment:
change to mprv_block or similar, _t is a reserved naming convention.

Reply:
Ah, didn’t know it was reserved. Good call!

Reply:
Changed to mprv_block.